test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: csi-driver-registrar
```

Comment:
this whole file should be updated to pull in the newest yamls:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/base/node.yaml

Reply:
specifically there are some updates around prestop lifecycle and possibly udev rules

Reply:
Done.